src/model/Board.js

```diff
@@ -142,23 +132,18 @@ class Board {
 	* @returns {Promise} A Promise that is resolved with the board added
 	*/
 	static addBoard(board) {
-		if (!board instanceof Board) {
+		if (!(board instanceof Board)) {
 			board = new Board(board);
 		}
 
-		return new Promise((resolve, reject) => {
-			if (board.GameID || board.Game) {
-				reject(new Error('Games cannot be added using this method; please use addGame() instead'));
-			} else {
-				resolve();
+		return Promise.resolve().then(() => {
```

Comment:
What's this about? Why not construct a promise?

Reply:
For my part, in my limited experience, I personally tend to prefer the `Promise.resolve().then(() => {` style unless it's in the case of wrapping a non-promise async library call that needs to be passed a callback.